product_docs/docs/hadoop_data_adapter/2/07_features_of_hdfs_fdw.mdx

```diff
@@ -24,6 +24,18 @@ Hadoop Foreign Data Wrapper supports aggregate pushdown. It pushes the aggregate
 
 For more information, see [Example: Aggregate pushdown](10b_example_aggregate_pushdown).
 
+## ORDER BY pushdown
+
+Hadoop Foreign Data Wrapper supports order by push-down. If possible, push `ORDER BY` clause to the remote server so that we get the ordered result set from the foreign server itself. It might help us to have an efficient merge join. 
+
+- [Example: ORDER BY pushdown](10c_example_order_by_push_down)
+
+## LIMIT OFFSET pushdown
+
+Hadoop Foreign Data Wrapper supports limit offset push-down. Wherever possible, perform LIMIT and OFFSET operations on the remote server. This reduces network traffic between local PostgreSQL and remote hdfs/Hive servers. ALL/NULL options are not supported on the Hive server, and thus they are not pushed down. Also, OFFSET without LIMIT is not supported on the remote server hence queries having that construct are not pushed.
```

Comment:
- Is this specific to PostgreSQL or should we be using Postgres here?
- Use consistent capitalization for Hive and hdfs. I suggest Hive and HDFS. 
- Use contractions: are not = aren't, is not = isn't

Reply:
changes done as per both comments